scripts/check_django_eol.py

```
@@ -0,0 +1,84 @@
+
+    # Example:
+    # 4.2.16 -> 4.2
+    cycle = ".".join(full_version.split(".")[:2])
+
+    return full_version, cycle
+
+
+def get_django_eol(cycle):
+    url = "https://endoflife.date/api/django.json"
+
+    response = requests.get(url)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)